### PR TITLE
conjure-up 2.3.0

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ilovezfs** [comment]
@battlemidget I'm not sure if I got the resources right here. There was a dependency conflict with PyYAML that was forcing it down to 3.11 despite the requirements.txt calling for 3.12. This may have to do with the fact that we need to use `--no-binary :all:` and some of versions of some the deps may not have an sdist.

File: Formula/conjure-up.rb [hunk 1]
```diff
@@ -43,23 +57,28 @@ class ConjureUp < Formula
   end
 
   resource "env" do
-    url "https://github.com/battlemidget/env/archive/1.0.0.tar.gz"
-    sha256 "a26b5c973df792ecfc1fc6b18cf696ccaf4c02c918b2878e81c6d495debaa340"
+    url "https://files.pythonhosted.org/packages/f4/67/97052e313f44543fadca265228f527c5394385916a13eb8b7932d7dad490/env-0.1.0.tar.gz"
```
Comment (battlemidget) [inline Formula/conjure-up.rb]: This will fail as it doesn't support python3, my port does

File: Formula/conjure-up.rb [hunk 2]
```diff
@@ -102,21 +131,21 @@ class ConjureUp < Formula
...
 
+  resource "pyvmomi" do
+    url "https://files.pythonhosted.org/packages/95/82/40f8c37a4c5264a2d581c24eb5f191cbdfe7f574d4013180edc84bbbf401/pyvmomi-6.5.0.2017.5-1.tar.gz"
+    sha256 "c28292594281901e894c39a0c06b4126a9c019b3d804c47fb116472299dbb42d"
+  end
+
   resource "PyYAML" do
-    url "https://files.pythonhosted.org/packages/4a/85/db5a2df477072b2902b0eb892feb37d88ac635d36245a72a6a69b23b383a/PyYAML-3.12.tar.gz"
-    sha256 "592766c6303207a20efc445587778322d7f73b161bd994f227adaa341ba212ab"
+    url "https://files.pythonhosted.org/packages/75/5e/b84feba55e20f8da46ead76f14a3943c8cb722d40360702b2365b91dec00/PyYAML-3.11.tar.gz"
```
Comment (battlemidget) [inline Formula/conjure-up.rb]: This should be ok, i think the feature set is the same between those releases

**Comment by ilovezfs** [comment]
@battlemidget the conflict on PyYAML is coming from https://github.com/juju/juju-bundlelib/issues/55. It was reported in January with no progress since. Is there any way not to use juju-bundlelib?

**Comment by battlemidget** [comment]
> @battlemidget the conflict on PyYAML is coming from juju/juju-bundlelib#55. It was reported in January with no progress since. Is there any way not to use juju-bundlelib?

I'll work with the team to get this resolved

**Comment by ilovezfs** [comment]
>I'll work with the team to get this resolved

Thanks @battlemidget :heart:

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by battlemidget** [comment]
@ilovezfs We got a new 0.5.5 juju-bundlelib uploaded to pypi now that fixes the PyYAML version pin

**Comment by ilovezfs** [comment]
@battlemidget wow that was quick! I've updated the PR accordingly.

**Pull request closed without merging**